astropy/tests/runner.py

```diff
@@ -170,7 +170,12 @@ def _generate_args(self, **kwargs):
         """
 
     _required_dependencies = ['pytest', 'pytest_remotedata', 'pytest_doctestplus', 'pytest_astropy_header']
-    _missing_dependancy_error = "Test dependencies are missing. You should install the 'pytest-astropy' package."
+    _missing_dependancy_error = (
+        "Test dependencies are missing: {module}. You should install the "
+        "'pytest-astropy' package (you may need to update the package if you "
+        "have a previous version installed, e.g., "
+        "`pip install pytest-astropy --upgrade` or the equivalent with conda).")
```

Comment:
This is not Sphinx, yo.
```suggestion
        "'pip install pytest-astropy --upgrade' or the equivalent with conda).")
```

Reply:
Oops, brain fart.